src/esp/sensor/Sensor.cpp

```diff
@@ -47,13 +47,29 @@ void SensorSpec::sanityCheck() {
 Sensor::Sensor(scene::SceneNode& node, SensorSpec::ptr spec)
     : Magnum::SceneGraph::AbstractFeature3D{node}, spec_(std::move(spec)) {
   node.setType(scene::SceneNodeType::SENSOR);
+  node.setSceneNodeTags({scene::SceneNodeTag::Leaf});
```

Comment:
Here, you need to CORRADE_ASSERT the node does NOT have any child before adding the tag.
`"Sensor::Sensor(): Cannot attach a sensor to a non-LEAF node. The number of children of this node is not zero."`